packages/net-misc/remmina/remmina.exlib

```diff
@@ -36,18 +41,19 @@ CMAKE_SRC_CONFIGURE_PARAMS=(
     -DWITH_GCRYPT=ON
     -DWITH_GETTEXT=ON
     -DWITH_LIBSECRET=ON
-    -DWITH_LIBSSH=ON
     -DWITH_TRANSLATIONS=ON
-    -DWITH_VTE=ON
-    -DWITH_ZLIB=ON
+    -DWITH_FREERDP=ON
+    -DWITH_LIBVNCSERVER=ON
 
     # off
     -DWITH_APPINDICATOR=OFF
+    -DWITH_TELEPATHY=OFF
+    # NB: upstream has marked avahi and spice as recommended
     -DWITH_AVAHI=OFF
-    -DWITH_FREERDP=OFF
     -DWITH_SPICE=OFF
-    -DWITH_SURVEY=OFF
```

Comment:
why is `WITH_SURVEY` removed?

Reply:
Upstream removed it. Please check rcgit17 here https://github.com/FreeRDP/Remmina/blob/next/CHANGELOG.md